### PR TITLE
Migrate to Github actions

### PR DESCRIPTION
**Pull request by BeryJu**
### Description

CLUSTERMAN-615

### Testing Done

Build fully works with Github actions, I had to do some minor adjustments to the Simulator since it was missing 2 attributes.

### CODE REVIEW

File: acceptance/moto/Dockerfile [hunk 1]
```diff
@@ -15,6 +15,8 @@ ADD . /moto/
 ENV PYTHONUNBUFFERED 1
 
 WORKDIR /moto/
+# Setuptools needs to be installed and up-to-date for install below
+RUN pip3 install -U setuptools pip
```
Comment (TheNavigat) [inline acceptance/moto/Dockerfile]: NIt: move these below, and pin the versions

File: clusterman/simulator/simulated_pool_manager.py [hunk 2]
```diff
@@ -42,6 +42,8 @@ def __init__(self, cluster: str, pool: str, configs: Sequence, simulator: "simul
         self.max_capacity = self.pool_config.read_int("scaling_limits.max_capacity")
         self.max_tasks_to_kill = read_int_or_inf(self.pool_config, "scaling_limits.max_tasks_to_kill")
         self.cluster_connector = SimulatedClusterConnector(self.cluster, self.pool, self.simulator)
+        self.max_weight_to_add = self.pool_config.read_int("scaling_limits.max_weight_to_add")
```
Comment (TheNavigat) [inline clusterman/simulator/simulated_pool_manager.py:45]: What are we doing with these? I don't see them being used anywhere

File: package/itest/ubuntu.sh [hunk 3]
```diff
@@ -34,9 +34,20 @@ apt-get install -y --force-yes python3.7 python3-pip python3-yaml awscli
 dpkg -i /dist/${DISTRIB_CODENAME}/clusterman_${PACKAGE_VERSION}_amd64.deb || true
 apt-get install -y --force-yes --fix-broken
 
+# Sometimes our acceptance tests run in parallel on the same box, so we need to use different CIDR ranges
+if [ "${DISTRIB_CODENAME}" = "xenial" ]; then
+    CIDR_BLOCK="10.0.0.0/24"
+else
+    CIDR_BLOCK="11.0.0.0/24"
```
Comment (TheNavigat) [inline package/itest/ubuntu.sh]: Are you sure about this? IIRC `11.0.0.0` isn't considered a private subnet

File: clusterman/simulator/simulated_pool_manager.py (hunk 2 above)
Comment (BeryJu) [reply clusterman/simulator/simulated_pool_manager.py:45]: The `simulate` command failed without these, I just copied them from the other pool manager.

File: package/itest/ubuntu.sh (hunk 3 above)
Comment (BeryJu) [reply package/itest/ubuntu.sh]: True, I also copied this block from `service-itest-runner`. I'll change it to at least use proper networks

File: clusterman/simulator/simulated_pool_manager.py (hunk 2 above)
Comment (TheNavigat) [reply clusterman/simulator/simulated_pool_manager.py:45]: I checked `git grep` and it looks like we're using it here:

`clusterman/autoscaler/pool_manager.py:            delta = min(self.max_weight_to_add, delta)`

Comment (BeryJu) [reply clusterman/simulator/simulated_pool_manager.py:45]: yeah the base pool manager expects this attribute to be set, same with `self.max_weight_to_remove`

File: acceptance/moto/Dockerfile [hunk 4]
```diff
@@ -15,16 +15,17 @@ ADD . /moto/
 ENV PYTHONUNBUFFERED 1
 
 WORKDIR /moto/
-# Setuptools needs to be installed and up-to-date for install below
-RUN pip3 install -U setuptools pip
+     # Setuptools needs to be installed and up-to-date for install of the actual packages
```
Comment (TheNavigat) [inline acceptance/moto/Dockerfile]: Nit: remove indentation

**Review by TheNavigat — APPROVED** [review]
LGTM. Please squash before merging

**Pull request merged**